libs/ts-rest/express/src/index.ts

```diff
@@ -1,3 +1,3 @@
 export * from './lib/ts-rest-express';
-export { TsRestRequest, TsRestRequestHandler } from './lib/types';
+export * from './lib/types';
```

Comment:
Hmm, can we instead expose particular types that make sense? 
By exposing all, this leaves room for future headache should we want do any internal refactoring, since that would potentially break downstream consuming clients.

Reply:
I see. how about only exposing `AppRouteImplementation` and `AppRouteOptions` types only? they would be useful for extracting individual handler functions.

Reply:
SGTM!